ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Python Developer with Flask - Javascript and SQL
+company: Ikas International (End client is Tier1 Investment Bank)
+url: https://www.ikasinternational.com/banking-jobs/python-developer-flask-sql-javascript-greenfield-670-day/1700878
+location: London, Canary Wharf
+          # Choose one of the following options
+contract: contract
+contact:
+    name: Fadi Harb
+    email: fharb@ikasinternational.com
+    phone: 02072209344
+    # <additional contact fields>: <additional value>
+created: !!timestamp 2018-04-25
+tags:
+  - Python
+  - SQL
+  - Flask
+  - Javascript
+  # include relevant tags
+---
+
+Python Developer - Flask, SQL, JavaScript
+
+Our client, a tier1 Investment Bank is currently seeking to hire a contract Python Developer to join a team that is currently building undertaking a new greenfield project to expand the capabilities of a current framework used for calculating risk.
+
+The role will sit with a Small team and will involve design, development and testing of developed applications. In order to be considered for this role, candidates must have the following skills:
```

Comment:
Can you say more about the role, for example, will the developer be expected to take part in a SCRUM type workflow, or something else? Are you looking for a junior developer or somebody with a great deal of experience? Clarity will help you find the right candidate.